mcsema/BC/Function.cpp

```diff
@@ -406,7 +560,15 @@ static bool TryLiftTerminator(TranslationContext &ctx,
             << "Function " << ctx.lifted_func->getName().str()
             << " calls " << targ_func->getName().str()
             << " at " << std::hex << inst.pc << std::dec;
-        InlineSubFuncCall(block, targ_func);
+        if ((ctx.cfg_inst->lp_ea == 0) || (ctx.lp_to_block[ctx.cfg_inst->lp_ea] == nullptr)) {
```

Comment:
What is this checking for?

Reply:
It checks if the call instruction has an associated `landingpad`. If yes, it uses invoke instead of call semantics. Only `ctx.cfg_inst->lp_ea == 0` check will be sufficient.